v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: false
+---
+
+<div class="filters filters-big clearfix">
+  <a href="create-security-certificates.html"><button style="width:28%" class="filter-button">Use Cockroach Cert</button>
```

Comment:
Since we're referring to commands, I think we should use the lowercase for `cockroach cert` and `openssl`. I don't think we can code-format inside a button, but the lowercase and bold (when selected) should be enough.